### ISSUE TITLE
GUI Filename List broken in 3.1.1

### ISSUE THREAD

**Issue by popy2k14** [nicoboss/nsz#50]
Hey.

Me again :-)
When i'll add two XCI files from icon view, the second is displayed wrong, wee picture.
But when i do some action on it (comp. / decomp.) the seems to be processed correctly.
Seems its just an cosmetic bug.

![XCI List broken](https://user-images.githubusercontent.com/6324597/71562853-9ad73200-2a86-11ea-8339-5a63e670daf9.PNG)

**Comment by nicoboss** [comment]
Thanks a lot for reporting this. There was more to it then we initially thought. Turned out if the path is too long to fit it will warp to the next line. However, because of its hardcoded height it wasn’t able to scale and so only displayed the last line. Fixing this was quite complex as then window is resizable and because of the way how Kivy handles on_size events inside a RecycleView (inside only visible widgets are loaded). Together with the major game list improvement I finally managed to fix this.
